RELEASE_NOTES.md

```diff
@@ -120,13 +120,17 @@ v1.0.0-alpha.X
   [#455](https://github.com/OpenAssetIO/OpenAssetIO/issues/455)
 
 - Migrated the following `ManagerInterface` methods to C++
-  `initialize`, `createState`, `createChildState`,
+  `initialize`, `managementPolicy`, `createState`, `createChildState`,
   `persistenceTokenForState`, `stateFromPersistenceToken`.
+  [#455](https://github.com/OpenAssetIO/OpenAssetIO/issues/455)
+  [#458](https://github.com/OpenAssetIO/OpenAssetIO/issues/458)
   [#445](https://github.com/OpenAssetIO/OpenAssetIO/issues/445)
```

Comment:
[consider][minor] Might be nice to sort these, and below?

Reply:
I sorted these in roughly the same order as the list of methods in the text.  Happy to sort in issue order though, if you prefer? 

In general, how should these be sorted?  Issue number order, completion order, or as here attempt to match the order with the text?

Reply:
You raise an excellent question! :) The OCD in me was probably just hanging after numeric, but totally happy to leave as-is